.github/workflows/scala-steward.yml

```diff
@@ -8,9 +8,9 @@ env:
 
 jobs:
   run_sbt_tests:
-    if: github.actor == 'scala-steward'
+    if: github.event.pull_request.head.repo.full_name == 'scala-steward/support-frontend'
```

Comment:
I think this is most likely the cause of the issue.

Reply:
👍  The difference should be that previously the condition was true for PRs when first raised but not when updated and with this it should be true for updated PRs too.  As it's now a 'what' test rather than a 'who'.  Unless the condition wasn't true for new PRs either, in which case I don't know what's going on.